docs/resources/resources-links-help.md

```diff
@@ -1,25 +1,76 @@
+| **Documentation gaps** | If the documentation is missing something that would help you understand how to create add-ins, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|
+
+### Report issues
+
+Submit issues and ask general questions related to add-ins, documentation, and samples using the community help channels.
+
+| **Issue type** | **Contact** |
+|:----------------------|:------------|
+| **Office Add-ins or Office JavaScript API issues** | If you encounter an issue (bug) with the Office JavaScript API, please submit an issue to the [Office JavaScript APIs GitHub repository](https://github.com/officedev/office-js/issues)|
+|**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|
+| **Issues with code samples** | Submit issues with samples to the [Office Add-ins code samples](https://github.com/OfficeDev/Office-Add-in-samples) repository. |
+
+### Documentation feedback
+
+Submit documentation feedback or updates using the community help channels.
+
+| **Feedback type** | **Contact** |
+|:--------------------------|:--------------------------|
+|**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|
+|**Documentation updates**| To make changes to the documentation yourself, choose the **Edit** link on an article and submit a pull request to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr)|
+
+### Feature request and general help
+
+Suggest a feature or vote up existing feature requests.
+
+| **Support need** | **Contact** |
+|:----------------------|:------------|
+|**Office Add-ins or Office JavaScript API feature requests**| To submit a feature request for the Office JavaScript API, please post your idea to the [Microsoft 365 Developer Platform Tech Community](https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform).|
+| **General questions** | Send general questions about the Office add-in platform to [Office Add-in Community Help](mailto:officeaddin. We encourage posting questions on the channels mentioned here and using email only if no other mode of communication is applicable. |
```

Comment:
Is "mailto:officeaddin" the right destination for the Community Help link? Does it need to be `mailto:officeaddin@microsoft.com` or something like that?

```suggestion
| **General questions** | Send general questions about the Office Add-ins platform to [Office Add-ins Community Help](mailto:officeaddin). We encourage posting questions on the channels mentioned here and using email only if no other mode of communication is applicable. |
```